.scalafmt.conf

```diff
@@ -2,3 +2,7 @@ version = "2.4.2"
 assumeStandardLibraryStripMargin = true
 align = none
 project.git = true
+
+project.excludeFilters = [
+  "(.*)/src/main/scala-3/(.*)"
```

Comment:
2.4.2 can't parse Scala 3 macros, and reformatting the whole codebase with 3+ is not an option right now

Reply:
It's fine to ignore this directory for now. Is there something specific blocking a Scalafmt upgrade?

Reply:
Not really, just when I reformatted with 3.0.5 all the files changed, so I thought I'd do the upgrade in a follow up :)